File: app/concepts/matestack/ui/core/form/checkbox/checkbox.rb
require_relative './base'

module Matestack::Ui::Core::Form::Checkbox
  class Checkbox < Base

    vue_js_component_name "matestack-ui-core-form-checkbox"

    def response
      div class: "matestack-ui-core-form-checkbox" do
        render_options
        render_errors
      end
    end

  end
end

File: app/concepts/matestack/ui/core/form/checkbox/checkbox.js
import Vue from "vue/dist/vue.esm";

import formCheckboxMixin from "./mixin";
import componentMixin from "../../component/component";

const componentDef = {
  mixins: [componentMixin, formCheckboxMixin],
  data() {
    return {};
  }
}

let component = Vue.component("matestack-ui-core-form-checkbox", componentDef);

export default componentDef;
